src/%IPM/Repo/Filesystem/Definition.cls

```diff
@@ -24,7 +24,6 @@ XData Commands
 {
 <commands>
 <command name="repo">
-<group name="Filesystem repository">
```

Comment:
Why is the group name deleted?

Reply:
(<group> tag no longer supported by %IPM.CLI.Commands.cls)